spark/unitycatalog/src/test/java/com/sparkuctest/UnityCatalogSupport.java

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.sparkuctest;
+
+import io.unitycatalog.client.ApiClient;
+import io.unitycatalog.client.api.CatalogsApi;
+import io.unitycatalog.client.api.SchemasApi;
+import io.unitycatalog.client.model.CreateCatalog;
+import io.unitycatalog.client.model.CreateSchema;
+import io.unitycatalog.server.UnityCatalogServer;
+import io.unitycatalog.server.utils.ServerProperties;
+import org.apache.log4j.Logger;
+import org.apache.spark.SparkConf;
+import org.junit.After;
+import org.junit.Before;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.nio.file.Files;
+import java.util.Properties;
+
+/**
+ * Abstract base class that provides Unity Catalog server integration for Delta tests.
+ *
+ * This class automatically:
+ * - Starts a local Unity Catalog server before all tests
+ * - Configures Spark to connect to the UC server
+ * - Stops the server and cleans up after all tests
+ *
+ * The UC server runs with Unity Catalog dependencies to provide catalog functionality
+ * for integration testing.
+ *
+ * Usage:
+ * <pre>
+ * public class MyUCTest extends UnityCatalogSupport {
+ *
+ *   {@literal @}Override
+ *   protected SparkConf getSparkConf() {
+ *     SparkConf conf = new SparkConf();
+ *     // ... configure spark ...
+ *     return configureSparkWithUnityCatalog(conf);
+ *   }
+ *
+ *   {@literal @}Test
+ *   public void myTest() {
+ *     // Use getUnityCatalogName() to reference the catalog
+ *     getSparkSession().sql("CREATE TABLE " + getUnityCatalogName() + ".default.test_table ...");
+ *   }
+ * }
+ * </pre>
+ */
+public abstract class UnityCatalogSupport {
```

Comment:
Can you hight the code in your intellij idea ?  I find I cannot track the code in intellij ...

Reply:
No keyword highlighted, and no code index and tracking.  I guess we need some change to the build.sbt.